src/api/app/views/webui2/webui/staging/projects/_checks.html.haml

```diff
@@ -0,0 +1,16 @@
+%td.p-2.w-25.font-weight-bold
+  %i.mr-2.fas{ class: icon_for_checks(checks, missing_checks) }
+  Checks
+%td.p-2
+  - if checks.blank? && missing_checks.blank?
+    None
+  - else
+    %ul
+      - missing_checks.each do |name|
+        %li
+          %span.check-pending
+            "#{name} (Expected - Waiting for status to be reported)"
```

Comment:
I think you don't need the double quotes

Reply:
done!